pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "tqdm_publisher"
+version="0.1.0"
+authors = [
+  { name="Garrett Flynn", email="garrett.flynn@catalystneuro.com" },
+  { name="Cody Baker", email="cody.baker@catalystneuro.com" },
+]
+description = "Subscribe to TQDM progress bars with arbitrary functions."
+readme = "README.md"
+requires-python = ">=3.8"
+license = {text = "BSD-3-Clause"}
```

Comment:
Great thing about this approach is we ought to be able to use the `file` specifier for this (and thereby point directly to the included LICENSE file, which you should ensure shows up in the build wheel so it ships with the package)

Reply:
Ah right. Good catch

Reply:
https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license details